packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -284,6 +327,40 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
           </Row>
         </>
       )}
+      {showRelOption && (
+        <CheckboxContainer>
+          <StyledText size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
+            {__('About this link.', 'web-stories')}
+          </StyledText>
+          {LinkTypes.map(({ key, title }) => (
+            <CheckboxWrapper key={key}>
+              <Checkbox
+                id={key}
+                name={key}
+                checked={link.rel && link.rel.includes(key)}
+                onChange={() => onChangeRel(key)}
+              />
+              <Label htmlFor={key}>
+                <Text
+                  size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+                  as="span"
+                >
+                  {title}
+                </Text>
+              </Label>
+            </CheckboxWrapper>
+          ))}
+
+          <Link
+            rel="noopener noreferrer"
+            target="_blank"
+            href={relHelpLink}
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+          >
+            {__('What are these?', 'web-stories')}
```

Comment:
As per https://github.com/google/web-stories-wp/issues/4440#issuecomment-956361256

```suggestion
            {__('Learn more', 'web-stories')}
```

Reply:
Fixed.